app/keepers.go

```diff
@@ -334,7 +334,7 @@ func (app *OsmosisApp) InitNormalKeepers(
 
 	app.SuperfluidKeeper = superfluidkeeper.NewKeeper(
 		appCodec, keys[superfluidtypes.StoreKey], app.GetSubspace(superfluidtypes.ModuleName),
-		*app.AccountKeeper, app.BankKeeper, app.StakingKeeper, app.DistrKeeper, app.EpochsKeeper, app.LockupKeeper, gammKeeper, app.IncentivesKeeper,
+		*app.AccountKeeper, app.BankKeeper, app.StakingKeeper, app.DistrKeeper, app.EpochsKeeper, app.LockupKeeper, app.GAMMKeeper, app.IncentivesKeeper,
```

Comment:
Changed this to be pointer based, since calling methods in gamm keeper in the old way couldn't read gammKeeper.hooks. Need additional reviews needed if changing this to pointer is safe

Reply:
Spooky that this wasn't always pointer based

Reply:
We need to double check that superfluid before did no action that triggered gamm hooks

Reply:
confirmed that it didnt